gnomad_constraint/plots/loading_functions.R

```diff
@@ -0,0 +1,285 @@
+library(conflicted)
+library(dplyr)
+library(gargle)
+library(googleAuthR)
+library(googleCloudStorageR)
+library(forcats, include.only = c("fct_recode"))
+library(purrr, include.only = c("map_df"))
+library(readr, include.only = c("read_tsv", "cols"))
+library(stringr, include.only = c("str_sub"))
+
+conflict_prefer("filter", "dplyr")
+conflict_prefer("select", "dplyr")
+conflict_prefer("mutate", "dplyr")
+conflict_prefer("%>%", "dplyr")
+
+####################################################################
+# Constants and functions related to data loading
+####################################################################
+data_versions <- c("v2", "v4")
+map_data_version <- list(
+  v2 = "2.1.1",
+  v4 = "4.1",
+  v4_tmp = "4.1"
+)
+
+default_output_path <- "."
+default_data_dir <- "data"
+default_plot_dir <- "plots"
+default_gene_lists <- "data/gene_lists"
+bucket <- "gs://gnomad"
+
+setup_directories <- function(
+    output_path = default_output_path,
+    data_dir = default_data_dir,
+    plot_dir = default_plot_dir,
+    gene_lists = default_gene_lists) {
```

Comment:
I tend to interpret output_path to mean the full path. How about these changes throughout:
output_path -> output_basedir
data_dir -> data_subdir 
plot_dir -> plot_subdir